survey/README.md

```
@@ -0,0 +1,4 @@
+## Quelques détails d'implémentation sur cette enquête
+
+- Support du pré-remplissage d'information, dont l'adresse de domicile, à partir du code d'accès
+- Présence d'échantillons partiels, sous le champ `ep` des réponses, pour déterminer s'il faut afficher ou non certaines questions/section. Ces échantillons sont: `exclusive`, une chaîne de caractère contenant le nom de l'échantillon partiel exclusif, `commonTrip` et `sameMode`, booléens mis à "vrai" si actifs. Ces échantillons sont déterminés aléatoirement en début d'enquête, lors de la validation du code d'accès.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)